### PR TITLE
WinUI-860408  prepared the Error Icon Feature Content in the WinUI MaskedTextBox UG Documentation

### PR DESCRIPTION
**Pull request by GilbertmanojSF3971**
Create the Error type heading in the WinUI MaskedTextBox UG and add the content for the below topics.
Error Types
Error Content
Custom Error Icon
Custom Error Border Brush

### CODE REVIEW

File: winui-toc.html [hunk 1]
```diff
@@ -382,6 +382,7 @@
             <li><a href="/winui/Masked-TextBox/customization">Customization</a></li>
             <li><a href="/winui/Masked-TextBox/events">Events</a></li>
             <li><a href="/winui/Masked-TextBox/how-to">How to</a></li>
+			<li><a href="/winui/Masked-TextBox/error-types">Error Types</a></li>
```
Comment (Manivannan-E) [inline winui-toc.html:385]: @GilbertmanojSF3971 Add new page above the customization with the heading error indication

Comment (GilbertmanojSF3971) [reply winui-toc.html]: @Manivannan-E Sir, Modified the UG documentation as per your concern.

File: winui/Masked-TextBox/Overview.md [hunk 2]
```diff
@@ -19,4 +19,5 @@ The [WinUI MaskedTextBox](https://www.syncfusion.com/winui-controls/masked-textb
 * `PromptChar`: Customize the prompt characters used to set the default characters as input in the absence of input.
 * `Value`: Enter values and clipboard operations in the control based on the ValueMaskFormat property.
 * `ValueMaskFormat`: Setting formatted values in the control using mask format values such as prompt, literals, and both.
-* `Customization`: Customize the UI of the masked text box.
\ No newline at end of file
+* `Customization`: Customize the UI of the masked text box.
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Overview.md:22]: @GilbertmanojSF3971 Remove Customization

Indicates errors or other information by displaying an icon and also show additional details when hovering the cursor over the icon.

Comment (GilbertmanojSF3971) [reply winui/Masked-TextBox/Overview.md:22]: Hi @Manivannan-E Sir. Customization title is already present in the control. And i have modified the Content for the Error Indication title.

File: winui/Masked-TextBox/Error-Indication.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+description: Explore the various error types supported by the Syncfusion WinUI MaskedTextBox (SfMaskedTextBox) control to enhance user experience and input validation.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Error Indication in WinUI MaskedTextBox
+
+The [WinUI MaskedTextBox](https://www.syncfusion.com/winui-controls/masked-textbox) control provides comprehensive error types support to facilitate effective input validation and enhance user experience. This feature enables users to utilize various error icons and styles based on specific validation scenarios.The following error types such as none, default, warning, information, critical, success, custom is available in the MaskedTextBox control.
+
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Error-Indication.md:13]: @GilbertmanojSF3971 
# Error Indication in WinUI MaskedTextBox

The MaskedTextBox control supports error indication by displaying an icon and providing additional details upon hovering over the icon. This powerful feature allows users to employ various error icons based on specific validation scenarios.

## Built-in Error Types:

The MaskedTextBox control offers several built-in error types:

None: No visible signals for errors.
Default: Red bottom border brush in the MaskedTextBox Control.
Warning: Visual highlight for potential concerns needing immediate attention.
Information: Share non-essential details or guidance.
Critical: Urgent issues requiring immediate action.
Success: Positive confirmation for valid input or successful actions.

File: winui/Masked-TextBox/Error-Indication.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI MaskedTextBox control with Errortype](MaskedTextBox_images/winui_masked_textbox_errortype.png)
+
+## Custom:
+
+The `Custom` error type in the SfMaskedTextBox allows users to define a unique error icon and customize the error border color using the `CustomErrorIcon` and `CustomErrorBorderBrush` properties, respectively.
+
+{% tabs %}
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Error-Indication.md:59]: @GilbertmanojSF3971 
## Custom error type:
The `Custom` error type provides users with the flexibility to customize both the error icon and the border brush. This customization is achieved through the following properties:

CustomErrorIcon: Allows you to set a custom error icon.
CustomErrorBorderBrush: Enables customization of the border brush for the `Custom` error type.

File: winui/Masked-TextBox/Error-Indication.md (hunk 3 above)
Comment (GilbertmanojSF3971) [reply winui/Masked-TextBox/Error-Indication.md:13]: @Manivannan-E Sir. Modified the description for Error Indication in WinUI MaskedTextBox Title.

File: winui/Masked-TextBox/Error-Indication.md (hunk 4 above)
Comment (GilbertmanojSF3971) [reply winui/Masked-TextBox/Error-Indication.md:64]: Hi @Manivannan-E Sir , Modified the description content for the Custom error type.

File: winui/Masked-TextBox/Error-Indication.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI MaskedTextBox control with Errortype Custom](MaskedTextBox_images/winui_masked_textbox_errortype_custom.png)
+
+## Error content:
+
+The `ErrorContent` property is used to display content when hovering over the error icon.
+
```
[the remark this answers is not in the document]
Comment (GilbertmanojSF3971) [reply winui/Masked-TextBox/Error-Indication.md:101]: Hi @Manivannan-E Sir, modified the content error content description as mentioned.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**